collision_buffer_distance: 0.1
waypoint_dist: 0.35
corridor_radius: 0.25

max_speed: 0.1
max_accel: 0.25

turtlebot_radius: 0.21
angular_col_res: 0.1
radial_col_res: 0.1
